docs/workshop-listing-guide.md

```diff
@@ -0,0 +1,97 @@
+# Guide to Listing Your Event on the ClojureBridge Website
+
+Everything you need to know about registering your event on https://clojurebridge.org .
```

Comment:
```suggestion
Everything you need to know about registering your event on https://clojurebridge.org.
```

The space might be intentional to avoid breaking the link. Ignore this if so